CHANGELOG.md

```diff
@@ -398,5 +398,6 @@ occasions, now it will be viable to play tall/defensively, without any requisite
 ## [5.0.0-beta.6] The Restless Dunes - TBA
 
 - Added new routes for Mannfred von Carstein: The Drakenhof Conclave
-- Now flavour is displayed in white colour and does not include a bullet point checkÇ
+- Now flavour is displayed in white colour and does not include a bullet point check
+- Goldtooth's third route objective to control regions has been fixed.
```

Comment:
Some times is not possible, but when possible use the keywords:

- **Added** for new features.
- **Changed** for changes in existing functionality.
- **Deprecated** for soon-to-be removed features.
- **Removed** for now removed features.
- **Fixed** for any bug fixes.
- **Security** in case of vulnerabilities.

Not a problem now tho.